src/EnergyPlus/Photovoltaics.cc

```diff
@@ -925,232 +940,220 @@ namespace Photovoltaics {
         //    integrated photovoltaics. Solar 2002, Sunrise on the Reliable Energy Economy, June 15-19, 2002 Reno, NV
 
         // Using/Aliasing
+        using PhotovoltaicThermalCollectors::GetPVTTsColl;
         using TranspiredCollector::GetUTSCTsColl;
 
         int ThisSurf; // working variable for indexing surfaces
         Real64 Ee;
 
         ThisSurf = state.dataPhotovoltaic->PVarray(PVnum).SurfacePtr;
+        auto &thisPVarray = state.dataPhotovoltaic->PVarray(PVnum);
```

Comment:
This is a very good use of a reference, thanks!

Reply:
> OK, so this does look close. I think the biggest red flags for me are:
> 
> * what appears to be a global variable still living in the code
> * an Array1D being passed by value, so a copy is being made (I realize it's going to be a small array, but we still don't want that pattern in new code)
> 
> @khaddad I am grateful you have been so responsive, and I'd like to do whatever I can to help get this in for this release. Are you able to address the final comments here, or do I need to do it to get this moving?
> 
> Thanks for this feature!

@Myoldmopar Thanks for the offer to help move this contribution along. Please go ahead and make the suggested changes.